packages/design-systems/src/components/composite/Datagrid/useDataGrid.tsx

```diff
@@ -19,16 +19,18 @@ export const useDataGrid = <TData extends Record<string, unknown>>({
   onFilterChange,
   localization,
   columnVisibility,
+  columnPinning,
   onColumnVisibilityChange,
   enableRowSelection = false,
   onRowSelectionChange,
   rowSelection,
 }: UseDataGridProps<TData>): DataGridInstance<TData> => {
   const { pageIndex = 0, pageSize = 10 } = pagination || {};
-  const [columnPinning, setColumnPinning] = useState<ColumnPinningState>({
-    left: ["select"],
-    right: [],
-  });
+  const [columnPinningState, setColumnPinningState] =
+    useState<ColumnPinningState>({
+      left: [...(columnPinning?.left || []), "select"],
```

Comment:
細かい点かつ、修正するかはお任せしますが、ここは以下のようにしたほうがselectが一番左に来るので良さそうに見えます。
```suggestion
      left: ["select", ...(columnPinning?.left || [])],
```

Reply:
<img width="272" alt="スクリーンショット 2024-03-11 11 43 36" src="https://github.com/tailor-platform/frontend-packages/assets/40710120/2afe57f8-a206-4f9f-8fd0-d402b1b41d2e">
<img width="391" alt="スクリーンショット 2024-03-11 11 42 28" src="https://github.com/tailor-platform/frontend-packages/assets/40710120/c2462bdc-355a-4de6-b87d-523f22893fd4">

Reply:
現状が左、修正案が右です。

Reply:
なるほど、たしかにこれは右の方がいいですね。